build/patches/Remove-some-params-from-search-url.patch

```diff
@@ -0,0 +1,76 @@
+From: uazo <uazo@users.noreply.github.com>
+Date: Sun, 6 Sep 2020 11:53:15 +0000
+Subject: remove aqs and other parameters from search urls
+
+---
+ components/search_engines/template_url.cc | 17 -----------------
+ 1 file changed, 17 deletions(-)
+
+diff --git a/components/search_engines/template_url.cc b/components/search_engines/template_url.cc
+--- a/components/search_engines/template_url.cc
++++ b/components/search_engines/template_url.cc
+@@ -643,13 +643,11 @@ bool TemplateURLRef::ParseParameter(size_t start,
+     if (!optional)
+       url->insert(start, kDefaultCount);
+   } else if (parameter == "google:assistedQueryStats") {
+-    replacements->push_back(Replacement(GOOGLE_ASSISTED_QUERY_STATS, start));
+   } else if (parameter == "google:baseURL") {
+     replacements->push_back(Replacement(GOOGLE_BASE_URL, start));
+   } else if (parameter == "google:baseSuggestURL") {
+     replacements->push_back(Replacement(GOOGLE_BASE_SUGGEST_URL, start));
+   } else if (parameter == "google:currentPageUrl") {
+-    replacements->push_back(Replacement(GOOGLE_CURRENT_PAGE_URL, start));
+   } else if (parameter == "google:cursorPosition") {
+     replacements->push_back(Replacement(GOOGLE_CURSOR_POSITION, start));
```

Comment:
This should be removed as well.

Reply:
it's related to omnibox suggestions.
just uncheck the settings

Reply:
Which settings would these be?